Lua/Lua.sublime-syntax

```diff
@@ -272,6 +272,7 @@ contexts:
         - function-arguments-meta
         - function-arguments
 
+    - include: variable-modifier
```

Comment:
Attributes are only legal in `local` declarations, so there is no need to add them to this context.

Reply:
In which context should I add them ? I added them in `infix-operator` so that the `<` in attributes is matched before `<` in infix operator.

Reply:
Probably introduce one that starts on `local`. That way it can't be confused with a comparison operator.